frontend/src/metabase-types/api/settings.ts

```diff
@@ -332,6 +345,7 @@ interface AdminSettings {
   "embedding-homepage": EmbeddingHomepageStatus;
   "setup-license-active-at-setup": boolean;
   "store-url": string;
+  gsheets: Partial<GdrivePayload>;
```

Comment:
Is this needed?

Reply:
needed? no, but it is part of settings, at least for now